### PR TITLE
Broken dependencies in 2.5

### PR DESCRIPTION
**Pull request by zikalino**
##### SUMMARY
This is minimal changes to fix broken modules in Ansible 2.5. 

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
azure_rm_containerinstance
azure_rm_mysqlserver
azure_rm_mysqldatabase
azure_rm_postgresqlserver
azure_rm_postgresqldatabase

##### ANSIBLE VERSION
2.5

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_containerinstance.py [hunk 1]
```diff
@@ -93,7 +93,7 @@
         description:
             - Force update of existing container instance. Any update will result in deletion and recreation of existing containers.
         type: bool
-        default: False
+        default: 'no'
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_containerinstance.py:96]: Why the false is incorrect?

Comment (zikalino) [reply lib/ansible/modules/cloud/azure/azure_rm_containerinstance.py:96]: not sure, just merged from newer version.... i think it can be removed

**Comment by mattclay** [comment]
Traceback in integration tests:

```
Traceback (most recent call last):
  File "/tmp/ansible_9k3JaP/ansible_module_azure_rm_mysqlserver.py", line 127, in <module>
    from azure.mgmt.rdbms.mysql import MySQLManagementClient
  File "/usr/local/lib/python2.7/dist-packages/azure/mgmt/rdbms/mysql/__init__.py", line 12, in <module>
    from .my_sql_management_client import MySQLManagementClient
  File "/usr/local/lib/python2.7/dist-packages/azure/mgmt/rdbms/mysql/my_sql_management_client.py", line 12, in <module>
    from msrest.service_client import SDKClient
ImportError: cannot import name SDKClient
```

**Comment by zikalino** [comment]
@mattclay i am trying to figure out what is causing that problem in the ci. i don't see this problem locally. anyway. i had to switch to some other tasks for a while, i will be back on it soon.

**Comment by yungezz** [comment]
@zikalino  could you pls look at  lint failure and test failure?

**Comment by yuwzho** [comment]
Hi @zikalino What is the status of this backport? Need close and reopen a new one?

**Comment by zikalino** [comment]
@yuwzho I will recheck this tonight. there's some problem with dependencies here. works localy but not via ci.

File: lib/ansible/modules/cloud/azure/azure_rm_mysqldatabase.py [hunk 2]
```diff
@@ -87,14 +87,14 @@
 import time
 from ansible.module_utils.azure_rm_common import AzureRMModuleBase
 
-try:
-    from msrestazure.azure_exceptions import CloudError
-    from msrestazure.azure_operation import AzureOperationPoller
-    from azure.mgmt.rdbms.mysql import MySQLManagementClient
-    from msrest.serialization import Model
-except ImportError:
+#try:
```
Comment (mattclay) [inline lib/ansible/modules/cloud/azure/azure_rm_mysqldatabase.py]: Restore the try/except block here.

File: lib/ansible/modules/cloud/azure/azure_rm_mysqlserver.py [hunk 3]
```diff
@@ -123,14 +123,14 @@
 import time
 from ansible.module_utils.azure_rm_common import AzureRMModuleBase
 
-try:
-    from msrestazure.azure_exceptions import CloudError
-    from msrestazure.azure_operation import AzureOperationPoller
-    from azure.mgmt.rdbms.mysql import MySQLManagementClient
-    from msrest.serialization import Model
-except ImportError:
+#try:
```
Comment (mattclay) [inline lib/ansible/modules/cloud/azure/azure_rm_mysqlserver.py]: Restore the try/except block here.

**Review by mattclay — CHANGES_REQUESTED** [review]

**Comment by zikalino** [comment]
@mattclay I have fixed the test & removed unnecessary changes. do you think we can merge it?

**Review by mattclay — APPROVED** [review]

**Comment by Fred-sun** [comment]
@yuwzho Could you help review this PR again? Thanks!

**Comment by zikalino** [comment]
@mattclay how about merging? could you merge? or somebody else should still take a look?

**Comment by Fred-sun** [comment]
@mattclay Could you help take look this PR ? Thanks!

**Comment by Fred-sun** [comment]
@mattclay Could you update the comment according by @zikalino's comment? Thanks!

**Comment by Fred-sun** [comment]
@mattclay Could you help check this PR? It can be merged or need somebody review ? Thanks!

**Comment by zikalino** [comment]
@mattclay could you merge this pr? it's hanging around for a while, and I am not sure who could merge this.....

**Pull request merged**